### DISCUSSION TITLE
Documentation prompt

### DISCUSSION THREAD

**Discussion by lDrakohh** [ispp-2324-ocial/KB#4] [Prompts IAS]
Insert here prompts use to generate documentation content

**Comment by lDrakohh** [comment]
Hemos usado un prompt para solicitar una plantilla para el team charter para poder trabajar sobre una base y definir los detalles del equipo de trabajo: https://chat.openai.com/share/64ae9bff-ce59-4c5b-b1c3-5ac1d6f88e3c

**Comment by lDrakohh** [comment]
Se ha pedido generar una plantilla para que los seceretarios puedan tener un documento común a todas las reuniones y podamos llevar al día todos los temas que se traten en las reuniones: https://chat.openai.com/share/29bb8de7-2318-426a-b361-ab166c764df2

**Comment by AitorRD** [comment]
Se ha utilizado un prompt para generar una plantilla para las incidencias en github: https://chat.openai.com/share/d19656ef-dfce-45ce-b91e-e472d18d578d

**Comment by adrrf** [comment]
se ha pedido que dadas varias clausulas injustas intente modificarlas para que estas no la sean https://chat.openai.com/share/7f4e006c-39ea-47df-8b8c-90f044ea758f

**Comment by adrrf** [comment]
Se ha pedido un draft para la política de privacidad de ocial https://chat.openai.com/share/32936bf1-0369-443b-b2fd-27860be7c74a

**Comment by lDrakohh** [comment]
Se ha utilizado Microsoft Copilot para generar las imágenes del StoryBoard para los clientes. Se han tenido que usar en total 30 prompts hasta conseguir las imágenes deseada. El tiempo estimado que ha llevado ha sido 1h 20 minutos. Es bastante tiempo, pero a mano nos hubiese llevaso 3-4h.

**Comment by paupenfer1** [comment]
Se ha pedido dar ideas sobre la creación de un contrato para usuarios pilotos de nuestra app https://chat.openai.com/share/6e535438-9cdd-4f39-8386-5707bf5706f6

**Comment by iniruimar** [comment]
He usado el siguiente prompt, para pasar documentos a md: https://chat.openai.com/share/b6b3987c-e5b3-443a-82ed-3ea3fe710754
